blueocean-pipeline-editor/src/main/js/components/editor/AddStepSelectionSheet.jsx

```diff
@@ -107,7 +107,7 @@ export class AddStepSelectionSheet extends Component<DefaultProps, Props, State>
                 </div>
                 <div className="editor-step-selector">
                 {stepMetadata && stepMetadata.filter(isStepValidForSelectionUI).filter(this.state.searchFilter).sort(stepSorter).map(step =>
-                    <div tabIndex="0" onKeyPress={e => this.selectItemByKeyPress(e, step)}
+                    <div tabIndex="0" data-functionName={step.functionName} onKeyPress={e => this.selectItemByKeyPress(e, step)}
```

Comment:
nice to see this pattern returning

Reply:
although should it use data-function-name vs hybrid case? (maybe @sophistifunk has a style opinion)

Reply:
`data-` should probably be viewed as a namespace, so what you put after it should be dictated by how you need to used it, and we don't want hyphens in our object prop names when we can avoid it.

Reply:
TL/DR: `data-functionName` is good. That's what I'd do. Therefore it must be good, of course. ∎

Reply:
ah good to know @sophistifunk 
![image](https://user-images.githubusercontent.com/14976/33046733-5ca3e034-cea6-11e7-99fc-1d0da6743327.png)